### DISCUSSION TITLE
Anyone have a good theme?

### DISCUSSION THREAD

**Discussion by xproot** [DSJAS/DSJAS#53] [General]
Need a theme, preferably one that's centered around red since I want something that looks like a Bank of America branch.

**Comment by SinisterSpatula** [comment]
Unless its a replica the scammers will hang up as soon as they see it.  Better to do random credit unions.  They look at bofa all day every day, they definitely know what its supposed to look like.

**Reply by xproot** [reply]
well do you have one?

**Comment by SinisterSpatula** [comment]
Not currently.  I'd work on some themes if Ethan feels up to making a YouTube video tutorial on how its done.  I didnt understand from just the existing documentation.
